apis/r/README.md

```diff
@@ -14,13 +14,26 @@ This code is hosted at R-universe, so you can do
 $ install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev', 'https://cloud.r-project.org'))
 ```
 
-## Note on building from source
+## From source
 
-Note that building from source (using `R CMD INSTALL .`) implies also building
-[libtiledbsoma](https://github.com/single-cell-data/TileDB-SOMA/tree/main/libtiledbsoma)
-from source so additional requirements listed in its
-[README.md](https://github.com/single-cell-data/TileDB-SOMA/blob/main/libtiledbsoma/README.md)
-may apply.
+### Requirements
+
+* Source installation requires the [`tiledb` R package](https://github.com/TileDB-Inc/TileDB-R) (which in turn depends on the [`tiledb` Core library](https://github.com/TileDB-Inc/TileDB)).
+* This, and the other R package dependencies, are listed in the [DESCRIPTION](https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/r/DESCRIPTION) file and can be installed via _e.g_ `remotes::install_dep(".")`.
+* In addition, the R package also depends on the [`libtiledbsoma` library](https://github.com/single-cell-data/TileDB-SOMA/tree/main/libtiledbsoma) -- which gets installed with the package as
+  described in the next section.
+
+### Step-by-step
+
+* Clone this repository: `git clone git@github.com:single-cell-data/TileDB-SOMA.git`
+* Change into the R API package directory: `cd TileDB-SOMA/apis/r`
+* Optionally, clean the file in to the repo: `./cleanup` (this is not needed the first time)
+* Optionally, update the `libtiledbsoma` sources: `./copy_source.sh` (which updates the includes tarball of `libtiledbsoma`).
```

Comment:
perhaps it would make sense to put the `remotes::install_deps(".")` bit here -- ?

Reply:
Dunno we have in the preceding para. I omitted it on purpose.